deploy/single/all-in-one-dbless-k4k8s-enterprise.yaml

```diff
@@ -646,7 +646,7 @@ spec:
           value: /dev/stderr
         - name: KONG_PROXY_ERROR_LOG
           value: /dev/stderr
-        image: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s:2.0.4.1-alpine
+        image: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition:2.2.0.0-alpine
```

Comment:
Why is this commit in this PR?

Reply:
Earlier commits that modified the contents of `deploy/manifests` preceded 3d44c9fe30d6d217ffb9a3216aa108dc59e03f32. This is the result of running `hack/build-single-manifests.sh` after merging that in from main.

Commits to the manifests under `deploy/single` are always going to be the sum result of the current contents of `deploy/manifests`. They shouldn't require human review (verify-manifests.sh checks them instead), but we do need to include them.